photon_stream/production/isdc/worker_node_status.py

```diff
@@ -14,7 +14,7 @@
 import numpy as np
 import json
 import fact
-from photon_stream.EventListReader import EventListReader
+from photon_stream.event_list_reader import event_list_reader
```

Comment:
This will fail. I am afraid this is not covered by unit tests. Here the class 'EventListReader' was renamed to the module name 'event_list_reader'